agentendpoint/inventory.go

```diff
@@ -28,6 +29,7 @@ func (c *Client) ReportInventory(ctx context.Context) {
 	state := inventory.Get(ctx)
 	write(ctx, state, inventoryURL)
 	c.report(ctx, state)
+	debug.FreeOSMemory()
```

Comment:
nit: can you mention this in the commit description too?

Reply:
ugh nice catch, I meant to remove this, but this means my testing was not as useful as I thought :(